src/main/java/uk/gov/pay/directdebit/webhook/gocardless/services/WebhookGoCardlessService.java

```diff
@@ -144,20 +177,4 @@ private void updateStateForPaymentEvents(List<GoCardlessEvent> eventsThatAffectP
                     return Optional.empty();
                 });
     }
-
-    private void logUnknownResourceTypeForEvent(GoCardlessEvent event) {
-        LOGGER.info("unhandled resource type for event with id {} ", event.getGoCardlessEventId());
-    }
-
-    private GoCardlessActionHandler getHandlerFor(GoCardlessResourceType goCardlessResourceType) {
-        switch (goCardlessResourceType) {
-            case PAYMENTS:
-            case PAYOUTS:
-                return goCardlessPaymentHandler;
-            case MANDATES:
-                return goCardlessMandateHandler;
-            default:
-                return this::logUnknownResourceTypeForEvent;
```

Comment:
Does this cause us to lose the logging for resource types we don’t expect to receive (resource types, not actions)?

Reply:
This will fixed in the next PR when I implement the "Make receipt of any GoCardless refund event log an error" and "Make receipt of any GoCardless subscription event log an error" acceptance criteria.